client/trino-jdbc/src/test/java/io/trino/jdbc/BaseTestJdbcResultSet.java

```diff
@@ -752,12 +752,12 @@ public void testArray()
             // no NULL elements
             checkRepresentation(connectedStatement.getStatement(), "ARRAY[1, 2]", Types.ARRAY, (rs, column) -> {
                 Array array = rs.getArray(column);
-                assertThat(array.getArray()).isEqualTo(new int[] {1, 2});
+                assertThat(array.getArray()).isEqualTo(new Object[] {1, 2});
```

Comment:
getArray returned array of objects (being integers). Previously this assertion worked but I guess that it was fixed in the AssertJ core - that's why I've made this change.